FiveCalls/FiveCalls/AboutSheet.swift

```diff
@@ -0,0 +1,140 @@
+                    AboutListItem(title: R.string.localizable.aboutItemWhyCall(), navigationLinkValue: WebViewContent.whycall)
+                    AboutListItem(title: R.string.localizable.aboutItemWhoWeAre(), navigationLinkValue: WebViewContent.whoweare)
+                    .navigationDestination(for: WebViewContent.self) { webViewContent in
+                        WebView(webViewContent: webViewContent)
+                            .navigationTitle(webViewContent.navigationTitle)
+                            .navigationBarTitleDisplayMode(.inline)
+                            .toolbarBackground(.visible)
+                            .toolbarBackground(Color.fivecallsDarkBlue)
+                            .toolbarColorScheme(.dark, for: .navigationBar)
+
+                    }
+                    AboutListItem(title: R.string.localizable.aboutItemFeedback()) {
+                        AnalyticsManager.shared.trackEventOld(withName: "Action: Feedback")
+                        if EmailComposerView.canSendEmail() {
+                            showEmailComposer = true
+                        } else {
+                            showEmailComposerAlert = true
+                        }
+                    }
+                    .sheet(isPresented: $showEmailComposer, content: {
+                        EmailComposerView() { _ in }
+                    })
+                    .alert(isPresented: $showEmailComposerAlert) {
+                        Alert(title: Text(R.string.localizable.cantSendEmailTitle()),
+                              message: Text(R.string.localizable.cantSendEmailMessage()),
+                              dismissButton: .default(Text(R.string.localizable.dismissTitle())))
+                    }
+                    AboutListItem(title: R.string.localizable.aboutItemShowWelcome()) {
+                        showWelcome = true
+                    }
+                    .sheet(isPresented: $showWelcome, content: {
+                        WelcomeView()
+                    })
+                }
+                
+                Section(header: Text(R.string.localizable.aboutSectionHeaderSocial()),
+                        footer: Text(R.string.localizable.aboutSectionFooterSocial()))
+                {
+                    AboutListItem(title: R.string.localizable.aboutItemFollowTwitter()) {
+                        followOnTwitter()
+                    }
+                    if appUrl != nil {
+                        AboutListItem(title: R.string.localizable.aboutItemShare(), url: appUrl)
+                    }
+                    AboutListItem(title: R.string.localizable.aboutItemRate()) { requestReview() }
+                }
+                
+                if let versionString {
+                    Section(
+                        header: HStack {
+                            Spacer()
+                            Text(versionString)
+                                .font(.caption)
```

Comment:
hmm, its applying uppercase even without that font applied. I'll see if I can figure out what's going on there...

Reply:
ah, it was because I was setting it as a header which I guess forces upper casing by default. Switched to footer like we had in the storyboard and it fixed it.